### ISSUE TITLE
can copy&paste new generated seed

### ISSUE THREAD

**Issue by dieeasy** [spesmilo/electrum#2594]
Using electrum 2.8.3 on linux.

While creating a new standard wallet and generating a new seed I then have to confirm it and looks like copy&paste (Ctrl+c, Ctrl+v) is disabled.
Anyway I can select the seed, click `Next`, then click the middle mouse button and it gets correctly pasted.

Seems to me X11 allows to bypass copy/paste protection and should be prevented as well.

**Comment by AbdussamadA** [comment]
yep and i like this feature/bug. let it be. most noobs don't know about middle mouse button copying and not allowing copying is mostly about stopping noobs. it's useful for creating test wallets!

**Comment by dieeasy** [comment]
@AbdussamadA I agree that's useful for tests but I don't expect such a "shortcut" on a mainnet wallet (perfectly fine on a testnet one, of course).

Even though middle-click knowledge or use might not be widespread, it still allows to bypass a security measure to prevent accidental funds loss. Looks very much like security through obscurity to me and I think that's plain wrong.

**Comment by fresheneesz** [comment]
What happens on my windows client seems like the right way to do it. You can copy your seed out, but when you click next, the clipboard is cleared, so if you want to paste it back in, you have to copy from whereever you stored it, and paste it in. I don't think copy/paste should be removed for this, but clearing the clipboard in between forces the user to actually verify their backup.

**Comment by dieeasy** [comment]
@fresheneesz as far as I know windows only has one clipboard and doesn't have one that gets automatically filled when you select and that you can paste from with the middle mouse button.

What I'm reporting about is the use of X11's [PRIMARY](https://tronche.com/gui/x/icccm/sec-2.html#s-2.6.1) "clipboard" to workaround clipboard paste prevention, so I don't see how this could be relevant under windows at all.

**Comment by fresheneesz** [comment]
@dieeasy I'm clarifying the behavior that exists under windows and recommending that the same behavior be replicated on the linux side.

**Comment by dieeasy** [comment]
@fresheneesz that's fine; thought you had not grasped the issue as you wrote "seems like the right way to do it". The linux client is also doing it right when it comes to the clipboard as far as I can tell. It's the primary that's apparently not handled the same way.

**Comment by ecdsa** [comment]
note: there is now a `debug_seed` option in the config, active only kivy. we could extend its use to Qt.
